ZipArchiverTest failes on Microsoft Windows

When building on Windows the ZipArchiverTest fails with the error 

```
[INFO] Running org.mule.tools.api.util.ZipArchiverTest
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0 s <<< FAILURE! - in org.mule.tools.api.util.ZipArchiverTest
[ERROR] toZipWithBlackListTest(org.mule.tools.api.util.ZipArchiverTest)  Time elapsed: 0 s  <<< ERROR!
java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/projects/mule-maven-plugin/mule-packager/target/test-classes/app-archiver/META-INF
        at org.mule.tools.api.util.ZipArchiverTest.setUp(ZipArchiverTest.java:34)

[ERROR] toZipTest(org.mule.tools.api.util.ZipArchiverTest)  Time elapsed: 0 s  <<< ERROR!
java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/projects/mule-maven-plugin/mule-packager/target/test-classes/app-archiver/META-INF
        at org.mule.tools.api.util.ZipArchiverTest.setUp(ZipArchiverTest.java:34)
```

--- a/mule-packager/src/test/java/org/mule/tools/api/util/ZipArchiverTest.java
+++ b/mule-packager/src/test/java/org/mule/tools/api/util/ZipArchiverTest.java
@@ -12,6 +12,7 @@ import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
 import java.io.File;
+import java.net.URI;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.zip.ZipFile;
@@ -31,7 +32,9 @@ public class ZipArchiverTest {
   @Before
   public void setUp() throws Exception {
     fileName = temp.getRoot().getAbsolutePath() + File.separator + "test-file.zip";
-    metaInfPath = Paths.get(getClass().getClassLoader().getResource("app-archiver/META-INF").toURI().getPath());
+    URI uri = getClass().getClassLoader().getResource("app-archiver/META-INF").toURI();
+    String mainPath = Paths.get(uri).toString();
+    metaInfPath = Paths.get(mainPath);
   }
 
   @Test

